app/benchmark_internal_test.go

```diff
@@ -48,3 +48,30 @@ func BenchmarkTopologyList(b *testing.B) {
 		topologyRegistry.renderTopologies(report, request)
 	}
 }
+
+func BenchmarkTopologyHosts(b *testing.B) {
+	benchmarkOneTopology(b, "hosts")
+}
+
+func BenchmarkTopologyContainers(b *testing.B) {
+	benchmarkOneTopology(b, "containers")
+}
+
+func benchmarkOneTopology(b *testing.B, topologyID string) {
+	report, err := loadReport()
+	if err != nil {
+		b.Fatal(err)
+	}
+	renderer, decorator, err := topologyRegistry.RendererForTopology(topologyID, url.Values{}, report)
+	if err != nil {
+		b.Fatal(err)
+	}
+	b.ReportAllocs()
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		b.StopTimer()
+		render.ResetCache()
+		b.StartTimer()
+		renderer.Render(report, decorator)
+	}
+}
```

Comment:
a `benchmarkRender(render func(report.Report))` helper would extract all the report loading and benchmarking machinery that is shared between this and `BenchmarkTopologyList`